### ISSUE TITLE
Chrome on the host system should not be a dependency of JenkinsX

### ISSUE THREAD

**Issue by xihaLong** [jenkins-x/jx#3987]
### Summary 
ChromeDP is included in a branch of code related to Git instantiation using the v1 variant of the ChromeDP API.
This means JX command line requires my machine to have Chrome installed. JX is generally for setting up lab environments on K8S clusters. This type of machine is more typically a server rather than consumer machine, where perhaps it makes less sense to install a consumer desktop web browser, if able to avoid it.

Open for discussion @wbrefvem @jstrachan 

Should JX be launching a browser? If so, lets put it in a container. If not, can remove this area of code for now

"v1"
v2 also does not have a "runner" package
https://github.com/jenkins-x/jx/blob/c9a4a473d3b4693fd0b48a8977c5f1ca8103bb11/pkg/jx/cmd/create_git_api_token.go#L11
Seems there is an overhaul of the ChromeDP API as mentioned in the Slack yesterday by Jeff Zhu:
https://github.com/chromedp/chromedp/commit/3d3bf22ccc321c3536c9c34fa5fefe7b575a6ba1

### Steps to reproduce the behavior
Source code inspection
https://github.com/jenkins-x/jx/blob/c9a4a473d3b4693fd0b48a8977c5f1ca8103bb11/pkg/jx/cmd/create_git_api_token.go#L10

### Expected behavior
Product installs all dependencies not mentioned in the manual, or does not require a compatible version of Chrome installed on the executing system
Is ChromeDP the right package to take as dependency here for firing up a browser or browser-equivalent? For something like obtaining a credential or creating the default token I would hope the related project will expose an API

Basically any solution other than taking a web browser as requirement will make sense.

### Actual behavior
Chrome is required by ChromeDP to function. This is not being encapsulated by running against a container with Chrome nor using an alternative dependency directly from GoLang to download and parse the Gitea site.

### Jx version
All since Feb 2018

The output of `jx version` is:
N/A

### Jenkins type
N/A

### Kubernetes cluster
N/A

### Operating system / Environment
Ubuntu 16.04.6 LTS
N/A
